src/steamship/invocable/lambda_handler.py

```diff
@@ -51,33 +51,33 @@ def _handler(
 
         if request and request.invocation:
             error_prefix = (
-                f"[ERROR - {request.invocation.http_verb} {request.invocation.app_path}] "
+                f"[ERROR - {request.invocation.http_verb} {request.invocation.invocation_path}] "
             )
         else:
             error_prefix = "[ERROR - ?VERB ?PATH] "
 
         try:
-            app = app_cls(client=client, config=request.invocation.config)
+            invocable = app_cls(client=client, config=request.invocation.config)
         except SteamshipError as se:
             return InvocableResponse.from_obj(se)
         except Exception as ex:
             logging.exception(ex)
             return InvocableResponse.error(
                 code=HTTPStatus.INTERNAL_SERVER_ERROR,
                 prefix=error_prefix,
-                message="Unable to initialize plugin/app.",
+                message="Unable to initialize plugin/package.",
```

Comment:
```suggestion
                message="Unable to initialize package/plugin.",
```

My OCD couldn't handle the inversion 8 lines below. 😱